mlir-tensorrt/test/Dialect/Plan/create-closed-regions.mlir

```diff
@@ -59,6 +69,21 @@ func.func @test_simple_shape_bound(%arg0: tensor<?x10xf32> {tensorrt.shape_profi
 //       CHECK:       yield %[[v4]] : tensor<?x10xf32>
 //       CHECK:     return %[[v2]] : tensor<?x10xf32>
 
+// CHECK-ALLOC-LABEL: @test_simple_shape_bound
+//  CHECK-ALLOC-SAME: (%[[arg0:.+]]: tensor<?x10xf32> {{.*}}) -> tensor<?x10xf32> {
+//       CHECK-ALLOC:     %[[c10:.+]] = arith.constant 10 : index
+//       CHECK-ALLOC:     %[[c0:.+]] = arith.constant 0 : index
+//       CHECK-ALLOC:     %[[dim:.+]] = tensor.dim %[[arg0]], %[[c0]] : tensor<?x10xf32>
+//       CHECK-ALLOC:     %[[v2:.+]] = plan.inline_closed_alloc_group target(#plan.tensorrt_cluster<disallow_shape_tensor_calculations = false, benefit = 1>)
+//       CHECK-ALLOC:      inputs(%[[arg0]], %[[dim]], %[[c10]] : tensor<?x10xf32>, index, index)
+//       CHECK-ALLOC:      in_attrs [#plan.bounds<shape, [1, 10], [40, 10]>, #plan.bounds<none>, #plan.bounds<none>]
+//       CHECK-ALLOC:      -> tensor<?x10xf32> {
+//       CHECK-ALLOC:     ^bb0(%[[in:.+]]: tensor<?x10xf32>, %[[in_1:.+]]: index, %[[in_2:.+]]: index):
+//       CHECK-ALLOC:       %[[v3:.+]] = stablehlo.exponential %[[in]] : tensor<?x10xf32>
+//       CHECK-ALLOC:       %[[v4:.+]] = with_shape %[[v3]](%[[in_1]], %[[in_2]]) :
+//       CHECK-ALLOC:       yield %[[v4]] : tensor<?x10xf32>
+//       CHECK-ALLOC:     return %[[v2]] : tensor<?x10xf32>
+
 // -----
```

Comment:
Include at least a `CHECK-ALLOC-LABEL` for the next test. Otherwise you don't know whether the `CHECK-ALLOC` statements in this last test are actually bleeding through the end of the test and matching the result of some other test segment. It is a common source of false positives that go undetected in large test files with multiple FileCheck prefixes in use.